src/livecodes/templates/starter/zig-wasm-starter.ts

```diff
@@ -0,0 +1,93 @@
+import type { Template } from '../../models';
+
+export const zigWasmStarter: Template = {
+  name: 'zig-wasm',
+  title: 'Zig (Wasm) Starter',
+  thumbnail: 'assets/templates/zig.svg',
```

Comment:
I think we can just call the starter template zig

```suggestion
  name: 'zig',
  title: 'Zig Starter',
```